src/vmspawn/vmspawn.c

```diff
@@ -1135,9 +1181,25 @@ static int run(int argc, char *argv[]) {
                          special_glyph(SPECIAL_GLYPH_LIGHT_SHADE), ansi_grey(), ansi_highlight(), ansi_grey(), ansi_normal());
         }
 
+        r = sd_listen_fds_with_names(true, &names);
+        if (r < 0)
+                return r;
```

Comment:
should log about errors, i.e. make this `log_error_errno(r, "Failed to get passed file descriptors: %m");` or so